lib/sisimai/bite/email/biglobe.rb

```diff
@@ -0,0 +1,148 @@
+module Sisimai::Bite::Email
+  module Biglobe
+    # Sisimai::Bite::Email::Biglobe parses a bounce email which created by BIGLOBE.
+    # Methods in the module are called from only Sisimai::Message.
+    class << self
+      # Imported from p5-Sisimail/lib/Sisimai/Bite/Email/Biglobe.pm
+      require 'sisimai/bite/email'
+
+      Re0 = {
+        :subject => %r/\AReturned mail:/,
+        :from    => %r/postmaster[@](?:biglobe|inacatv|tmtv|ttv)[.]ne[.]jp/,
+      }
+      Re1 = {
+        :begin  => %r/\A   ----- The following addresses had delivery problems -----\z/,
+        :error  => %r/\A   ----- Non-delivered information -----\z/,
+        :rfc822 => %r|\AContent-Type: message/rfc822\z|,
+        :endof  => %r/\A__END_OF_EMAIL_MESSAGE__\z/,
+      }
+      ReFailure = {
```

Comment:
__[rubocop]__
* Style/MutableConstant: Freeze mutable objects assigned to constants.

<!-- Comment by [SideCI](https://sideci.com) -->